mmv1/products/compute/terraform.yaml

```diff
@@ -900,6 +900,26 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       post_create: templates/terraform/post_create/labels.erb
   GlobalForwardingRule: !ruby/object:Overrides::Terraform::ResourceOverride
     examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "external_ssl_proxy_lb_mig_backend_custom_header"
+        primary_resource_id: "default"
```

Comment:
If we are confident in this sample working as-is, we can move forward without testing by adding
`skip_test: true` here.

Reply:
@c2thorn  - Thank you. I have tested this script several time, so no issues for now. Can remove this `skip_test: true` after sometime ?